terms/neuro/modelSystemName.json

```diff
@@ -199,7 +199,7 @@
         	"source": "https://www.jax.org/strain/31950"
         },
         {
-        	"const": "PicalmH458R_Aduci",
+        	"const": "Picalm*H458R",
```

Comment:
I have some concerns with the asterisk because asterisks have special meanings at times, e.g. regular expressions.  Can it be left out?